lib/WebService/Async/SmartyStreets.pm

```diff
@@ -0,0 +1,128 @@
+package WebService::Async::SmartyStreets;
+# ABSTRACT: Access SmartyStreet API
+
+use strict;
+use warnings;
+
+our $VERSION = '1.001';
+
+=pod
+
+WebService::Async::SmartyStreets
```

Comment:
Look at other modules for how to do this, https://github.com/binary-com/perl-Date-Utility/blob/master/lib/Date/Utility.pm#L8 for example:

- `NAME` should include a brief line similar to abstract
- `SYNOPSIS` has examples
- `DESCRIPTION` has more details about the module, the service and how to use it